Installation Problem

Hi,

When i run my project, I get the following errors

 ```
    /....../flutter/.pub-cache/hosted/pub.dartlang.org/hardware_buttons-0.2.1/ios/Classes/VolumeButtenStreamHandler.swift:43:61: error: 'UIApplicationDidBecomeActive' has been renamed to 'UIApplication.didBecomeActiveNotification'
            notificationCenter.removeObserver(Notification.Name.UIApplicationDidBecomeActive)
                                                                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                UIApplication.didBecomeActiveNotification
    UIKit.NSNotification.Name:11:23: note: 'UIApplicationDidBecomeActive' was obsoleted in Swift 4.2
        public static let UIApplicationDidBecomeActive: NSNotification.Name
                          ^
    /....../flutter/.pub-cache/hosted/pub.dartlang.org/hardware_buttons-0.2.1/ios/Classes/VolumeButtenStreamHandler.swift:49:42: error: 'AVAudioSessionCategoryAmbient' has been renamed to 'AVAudioSession.Category.ambient'
                try audioSession.setCategory(AVAudioSessionCategoryAmbient)
                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                             AVAudioSession.Category.ambient
    AVFoundation.AVAudioSessionCategoryAmbient:3:12: note: 'AVAudioSessionCategoryAmbient' was obsoleted in Swift 3
    public let AVAudioSessionCategoryAmbient: AVAudioSession.Category
```

Do you have an idea ?

Thanks

--- a/example/ios/Runner.xcodeproj/project.pbxproj
+++ b/example/ios/Runner.xcodeproj/project.pbxproj
@@ -182,7 +182,7 @@
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
 						DevelopmentTeam = NF7D3MF6YF;
-						LastSwiftMigration = 0910;
+						LastSwiftMigration = 1030;
 					};
 				};
 			};
@@ -391,7 +391,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = flutter.moum.hardwareButtonsExample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 4.2;
 				VERSIONING_SYSTEM = "apple-generic";
 			};
 			name = Profile;
@@ -526,7 +526,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
 				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 4.2;
 				VERSIONING_SYSTEM = "apple-generic";
 			};
 			name = Debug;
@@ -553,7 +553,7 @@
 				PRODUCT_BUNDLE_IDENTIFIER = flutter.moum.hardwareButtonsExample;
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				SWIFT_OBJC_BRIDGING_HEADER = "Runner/Runner-Bridging-Header.h";
-				SWIFT_VERSION = 4.0;
+				SWIFT_VERSION = 4.2;
 				VERSIONING_SYSTEM = "apple-generic";
 			};
 			name = Release;

--- a/ios/Classes/SingleHomeAndLockObserver.swift
+++ b/ios/Classes/SingleHomeAndLockObserver.swift
@@ -75,7 +75,7 @@ class SingleHomeAndLockObserver {
         notificationCenter.addObserver(
             self,
             selector: #selector(applicationWillResignActive),
-            name: Notification.Name.UIApplicationWillResignActive,
+            name: UIApplication.willResignActiveNotification,
             object: nil)
     }
     
@@ -89,7 +89,7 @@ class SingleHomeAndLockObserver {
         
         // Home Button
         notificationCenter.removeObserver(
-            Notification.Name.UIApplicationWillResignActive)
+            UIApplication.willResignActiveNotification)
     }
     
     // Lock Button Detection

--- a/ios/Classes/VolumeButtenStreamHandler.swift
+++ b/ios/Classes/VolumeButtenStreamHandler.swift
@@ -32,7 +32,7 @@ public class VolumeButtenStreamHandler: NSObject, FlutterStreamHandler {
         notificationCenter.addObserver(
             self,
             selector: #selector(activateAudioSession),
-            name: Notification.Name.UIApplicationDidBecomeActive,
+            name: UIApplication.didBecomeActiveNotification,
             object: nil)
     }
     
@@ -40,13 +40,13 @@ public class VolumeButtenStreamHandler: NSObject, FlutterStreamHandler {
     private func removeVolumeObserver() {
         notificationCenter.removeObserver(self,
                                           forKeyPath: "outputVolume")
-        notificationCenter.removeObserver(Notification.Name.UIApplicationDidBecomeActive)
+        notificationCenter.removeObserver(UIApplication.didBecomeActiveNotification)
     }
     
     @objc func activateAudioSession(){
         let audioSession = AVAudioSession.sharedInstance()
         do {
-            try audioSession.setCategory(AVAudioSessionCategoryAmbient)
+            try audioSession.setCategory(AVAudioSession.Category.ambient)
             try audioSession.setActive(true)
             audioSession.addObserver(self,
                                      forKeyPath: "outputVolume",

